Xamarin.Forms.Platform.iOS/Renderers/MediaElementRenderer.cs

```diff
@@ -131,8 +131,8 @@ protected override void Dispose(bool disposing)
 
 			RemoveStatusObserver();
 
-			_avPlayerViewController?.Player?.Pause();
-			_avPlayerViewController?.Player?.ReplaceCurrentItemWithPlayerItem(null);
+			_avPlayerViewController?.Dispose();
```

Comment:
This one will call before on element changed. maybe it will throw exceptions on ElementChange function